public/index.html

```diff
@@ -9,6 +9,72 @@
   <title>Tech Quiz</title>
 </head>
 <body>
+  <main class="container">
+    <h1> Take a Quiz </h1>
+    <form action="" method=""> 
+        <div class="category-container">
+            <h4>Choose Category :</h4>
+            <div class="chooses">
+                <div class="choose">
+                    <input id="HTML" type="radio" name="category" value="HTML">
+                    <label for="HTML">HTML</label>
+                </div>
+                <div class="choose">
+                    <input id="JavaScript" type="radio" name="category" value="JavaScript">
+                    <label for="JavaScript">JavaScript</label>
+                </div>
+                <div class="choose">
+                    <input id="BASH" type="radio" name="category" value="BASH">
+                    <label for="BASH">BASH</label>
+                </div>
+                <div class="choose">
+                    <input id="Linux" type="radio" name="category" value="Linux">
+                    <label for="Linux">Linux</label>
+                </div>
+                <div class="choose">
+                    <input id="MySQL" type="radio" name="category" value="MySQL">
+                    <label for="MySQL">MySQL</label>
+                </div>
+                <div class="choose">
+                    <input id="DevOps" type="radio" name="category" value="DevOps">
+                    <label for="DevOps">DevOps</label>
+                </div>
+                <div class="choose">
+                    <input id="WordPress" type="radio" name="category" value="WordPress">
+                    <label for="WordPress">WordPress</label>
+                </div>
+                <div class="choose">
+                    <input id="Kubernetes" type="radio" name="category" value="Kubernetes">
+                    <label for="Kubernetes">Kubernetes</label>
+                </div>
+                <div class="choose">
+                    <input id="Docker" type="radio" name="category" value="Docker">
+                    <label for="Docker">Docker</label>
+                </div>
+            </div>
+        </div>
+
+        <div class="Difficulty-container">
+            <h4>Choose Difficulty :</h4>
+            <div class="chooses">
+                <div class="choose">
+                    <input id="Easy" type="radio" name="Difficulty" value="Easy">
+                    <label for="Easy">Easy</label>
+                </div>
+                <div class="choose">
+                    <input id="Medium" type="radio" name="Difficulty" value="Medium">
+                    <label for="Medium">Medium</label>
+                </div>
+                <div class="choose">
+                    <input id="Hard" type="radio" name="Difficulty" value="Hard">
+                    <label for="Hard">Hard</label>
+                </div>
+            </div>
+        </div>
+        <button class="start-button">Start Quiz</button>
+    </form>
+</main>
+
```

Comment:
Focus on writing the ids, classes, and value of for attributes in lowercase.
Switch all the div elements to section elements.